Libraries/Image/Image.android.js

```diff
@@ -193,7 +193,7 @@ var Image = React.createClass({
     getSize(
       url: string,
       success: (width: number, height: number) => void,
-      failure: (error: any) => void,
+      failure?: (error: any) => void,
```

Comment:
We already handle the case where this is undefined, the flow type wasn't up to date.